app/src/main/java/com/vero/tasky/core/data/local/UserPreferencesImpl.kt

```diff
@@ -38,7 +38,6 @@ class UserPreferencesImpl @Inject constructor(
             putString(UserPreferences.TOKEN, user.token)
             putString(UserPreferences.FULL_NAME, user.fullName)
             putString(UserPreferences.USER_ID, user.userId)
-            apply()
         }
```

Comment:
if I use pref.edit, it will be called from the API by itself 
![image](https://github.com/AndreVero/Tasky/assets/39765531/a564aa2c-c990-4bab-9601-0f5a47d19d04)

Reply:
Oh interesting
Wasnt aware of that